super0.c

```diff
@@ -923,12 +926,35 @@ static int store_super0(struct supertype *st, int fd)
 	return 0;
 }
 
+static __u64 avail_size0(struct supertype *st, __u64 devsize,
```

Comment:
please move function body up.
I know that we will destroy blame, but commit message you added it has nothing interesting so I think it is fine.

eventually, you can probably call:
`st->ss->avail_size`